src/main/markdown/makinggwtbetter.md

```diff
@@ -514,51 +511,24 @@ Please do submit code. Here's what you need to do:
 
 1.  Decide which code you want to submit. A submission should be a set of changes that addresses one issue in the [GWT issue tracker](https://github.com/gwtproject/gwt/issues?q=is%3Aissue). Please don't mix more than one logical change per submittal, because it makes the history hard to follow. If you want to make a change that doesn't have a corresponding issue in the issue tracker, please create one.
 2.  Also, coordinate with team members that are listed on the issue in question. This ensures that work isn't being duplicated and communicating your plan early also generally leads to better patches.
-3.  Ensure that your code adheres to the [GWT source code style](#codestyle).
-4.  Ensure that there are unit tests for your code.
-5.  Sign in to [GWT's Gerrit service](https://gwt-review.googlesource.com) and sign the [Contributor License Agreement](https://gwt-review.googlesource.com/#/settings/agreements).
-6.  Push your changes to Gerrit for review. The first time you will need to do some [initial setup](#gerritsetup). Follow Gerrit's documentation to create a change and upload it for review.
-7.  After setting up gerrit on your system, you can push your changes for review using:
-      `git push origin HEAD:refs/for/master`
-
-### Gerrit setup<a id="gerritsetup"></a>
-
-Finally, to actually create an issue is slightly more involved, but most of the steps only need to be done once:
-
-1. Setup your HTTP Password: 
-   Go to Settings in Gerrit. Under Settings you will find "HTTP Password" and click "Obtain Password"
-   and follow the steps to get your HTTP Password and/or to setup your .netrc file.
-2. Setup your Gerrit commit-msg hook (optional, but recommended): 
-   Gerrit provides a commit hook at `https://gwt-review.googlesource.com/tools/hooks/commit-msg` 
-   to automatically add Change-Id lines to your commits.  
-   Download this and add it to your checkout's .git/hooks directory: 
-   `curl -o .git/hooks/commit-msg https://gwt-review.googlesource.com/tools/hooks/commit-msg && chmod +x .git/hooks/commit-msg`
-3. Make a change and commit it locally using git (e.g., edit a file foo and then run `git commit -m "my first change" foofile`).
-   Push the commit to Gerrit for review: `git push origin HEAD:refs/for/master`.
-
-If you have any problems you might want to consult the gerrit documentation on [Uploading Changes](https://gerrit-review.googlesource.com/Documentation/user-upload.html) 
+3.  Make a fork of the [GWT repository](https://github.com/gwtproject/gwt)
```

Comment:
What do you think about "..of the appropriate repository (link to github.com/gwtproject or https://github.com/orgs/gwtproject/repositories)"?